newspaper/images.py

```diff
@@ -56,8 +56,7 @@ def image_entropy(img):
 
 
 def square_image(img):
-    """If the image is taller than it is wide, square it off. determine
-    which pieces to cut off based on the entropy pieces
+    """If the height of the image is greater than it is width, then a square image is returned. Pieces to cut is based on the entropy pieces
```

Comment:
Change 'greater than it is width' to 'greater than its width' and I will accept this PR.

Reply:
Please put 'Pieces to cut is based on the entropy pieces' on a new line